app/docker/cothority/Dockerfile

```diff
@@ -24,7 +24,6 @@ RUN echo 'export GOPATH=/home/dedis/go' >> ~/.profile; \
     echo 'export PATH=$PATH:/usr/lib/go-1.6/bin:~/bin:$GOPATH/bin' >> ~/.bashrc; \
```

Comment:
If I do what is explained in the readme. It just says that there is no docker daemon running. Can you run it quickly?